net-im/toxic/toxic-0.16.0.ebuild

```diff
@@ -115,3 +122,13 @@ src_install() {
 		rm -r "${ED}"/usr/share/${PN}/sounds || die "Could not remove sounds directory"
 	fi
 }
+
+pkg_postinst() {
+	xdg_desktop_database_update
+	einfo	"An example config file has been installed at /usr/share/toxic/toxic.conf.example"
+	einfo	"Feel free to copy it to ~/.config/tox/toxic.conf and modify as desired"
+}
+
+pkg_postrm() {
+	xdg_desktop_database_update
+}
```

Comment:
I believe the update does not need to be specified here, as it will come by default via `inherit xdg`, but for the `postinst` it is needed since it is specified here (for the `einfo`).

That written, I don't think it hurts to have it either though.